apps/src/templates/studioHomepages/StudentSections.jsx

```diff
@@ -10,6 +10,7 @@ export default class StudentSections extends Component {
     initialSections: PropTypes.array.isRequired,
     isRtl: PropTypes.bool.isRequired,
     canLeave: PropTypes.bool.isRequired,
+    isTeacher: PropTypes.bool
```

Comment:
`isTeacher` is passed around to lots of components for different reasons so I included a comment explaining its use in this context.